schemas/telemetry/configuration-schema.json

```diff
@@ -288,6 +288,14 @@
                   "type": "integer",
                   "description": "The upload frequency of batches (in milliseconds)"
                 },
+                "batch_processing_level": {
+                  "type": "integer",
+                  "description": "Maximum number of batches processed sequencially without a delay"
+                },
+                "background_tasks_enabled": {
+                  "type": "boolean",
+                  "description": "Whether UIApplication background tasks are enabled"
```

Comment:
this needs to be changed, it is not only `UIApplication` :)

Reply:
I think it's a configuration only available on iOS 🤔

Reply:
yeah, I was about to ask what's that property all about ?

Reply:
This is something that we forgot to add the other day and I discovered it when working on `batch_processing_level`

I can move it to a separate PR too :)